packages/core/src/hooks/useTokenPrice.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from 'react'
+
+import { getTokenPrice, adapterCoinGecko } from '../helpers/price'
+
+export const useTokenPrice = (token: string, quote: string, adapter = adapterCoinGecko) => {
```

Comment:
since useTokenBalance accepts a address of a token i feel like useTokenPrice should also take address instead of token id

Reply:
Possibly, there should be a way to map token address(or symbol) to the token_id required to interact with coingecko...